src/main/java/website/frontrow/artificial/intelligence/ArtificialIntelligence.java

```diff
@@ -47,44 +48,41 @@ public void aiMover()
 	{
 		this.players = this.level.getPlayers();
 		this.units = this.level.getUnits();
-		
-		for(Unit u : units)
-		{
-			if (u instanceof Enemy)
-			{
-					this.enemies.add((Enemy) u);
-			}		
-		}
+
+		// Abbreviation u here to adhere to lambda expressions.
+		this.enemies.addAll(units.stream().filter(
+				u -> u instanceof Enemy).map(
```

Comment:
That's not a case of indenting, that's a style decision to keep it readable.

<layout